content/en/blog/2023/experimental-dual-stack/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Support for Dual Stack Kubernetes Clusters"
+description: "Experimental support for Dual Stack Kubernetes Clusters."
+publishdate: 2023-02-20
+attribution: "Steve Zhang (Intel), Alex Xu (Intel), Iris Ding (Intel), Jacob Delgado (F5), Ying-chun Cai (formerly F5)"
+keywords: [dual-stack]
+---
+
+Over the past year, both Intel and F5 have collaborated on an effort to bring support for
+[Kubernetes Dual-Stack networking](https://kubernetes.io/docs/concepts/services-networking/dual-stack/) to Istio.
+
+## Background
+
+The journey has taken us longer than anticipated and we continue to have work to do. The team initially started with a design based
+on a reference implementation from F5. The design led to an [RFC](https://docs.google.com/document/d/1oT6pmRhOw7AtsldU0-HbfA0zA26j9LYiBD_eepeErsQ/edit?usp=sharing) that
+caused us to re-examine our approach. Notably, there were concerns about memory and performance issues that the community wanted
+to be addressed before implementation.
+
+## Experimental Dual Stack branch
+
+While the team was re-evaluating the options, we created a new branch, [experimental-dual-stack]({{< github_raw >}}/tree/experimental-dual-stack),
+to experiment and implement the reference design. We will detail how to use and install artifacts
+from that branch in an upcoming blog, however, be aware that its initial intent was to learn how to approach development
+in a way that we can implement a big impacting feature within Istio without causing regressions to single stack clusters.
+It was also branched between Istio 1.13 and 1.14 and hasn't kept up to date with the master branch. Using artifacts built
+from this branch would be considered highly experimental. While the branch builds locally, it fails the CI pipeline when
+another PR is added. With that said, there are still people and companies that are using artifacts from this branch
+in both staging and production environments.
```

Comment:
This blog leaves me wondering should I be using only 1.17 or only this experimental branch or maybe it should depend on my use case?   I had a similar set of questions (master vs. experimental branch) when trying to do some dual-stack testing recently.   Assuming you want nearly everyone using 1.17 then maybe just change the wording here to indicate this is historical and remove any references to people using the artifacts.   That can be left unsaid.